exercises/pangram/src/example.c

```diff
@@ -0,0 +1,29 @@
+#include "pangram.h"
+#include <string.h>
+
+#define NUMBER_OF_LETTERS (26)
+#define DOWNCASE(letter) ((letter >= 'A') && (letter <= 'Z')) ? letter + ('a' - 'A') : letter
+
+bool is_pangram(const char *sentence) {
+   int sentence_length = strlen(sentence);
+   char found_letters[NUMBER_OF_LETTERS];
+   int i;
+
+   memset(found_letters, 0, sizeof(found_letters));
+
+   for(i = 0; i < sentence_length; i++) {
+      char letter = DOWNCASE(sentence[i]);
+
+      if((letter >= 'a') && letter <= 'z') {
```

Comment:
Inconsistent use of parens (included on the first check, omitted on the second).